rice/flags.go

```diff
@@ -22,6 +22,8 @@ var flags struct {
 	EmbedGo   struct{} `command:"embed-go" alias:"embed"`
 	EmbedSyso struct{} `command:"embed-syso"`
 	Clean     struct{} `command:"clean"`
+
+	Tags []string `long:"tags" description:"Tags to use"`
```

Comment:
The description should be more clear; Tags to use where?

Reply:
Tags to use with the implicit go build ?